modules/app/src/main/java/org/locationtech/jtstest/function/SpatialIndexFunctions.java

```diff
@@ -117,6 +117,14 @@ public static Geometry strTreeNN(Geometry geoms, Geometry geom)
     return (Geometry) result;
   }
 
+  public static Geometry[] strTreeNNk(Geometry geoms, Geometry geom)
```

Comment:
@dr-jts I just try to add "int k" as a parameter. But it seems TestBuilder UI doesn't take more than 2 parameters. It only has two input areas. Not sure about this now.